### PR TITLE
US_03.06 | New Item > Create Multiconfiguration project

### PR DESCRIPTION
**Pull request by emmashabsis**
https://trello.com/c/2AZJytiZ/12-us0306-new-item-create-multiconfiguration-project

### CODE REVIEW

File: cypress/e2e/multiconfigProject1.cy.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/// <reference types="cypress"/>
+describe('multiconfigProject1', () => {
+
+    it('Create multiconfigProject1', function() {
```
Comment (KravchukKseniya) [inline cypress/e2e/multiconfigProject1.cy.js:4]: Please, use the title of your test case from Trello for test name.

**Comment by KravchukKseniya** [comment]
Please, add a link to your test case, not to US

**Comment by emmashabsis** [comment]
Thank you. I am trying to understand, how and what to do. I did not to do
anything for Trello. I need more time to study and check.

File: cypress/e2e/multiconfigProject1.cy.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        cy.get('a[href="newJob"]').click()
+        cy.get('input#name').type('Project1')
+        cy.get('li[tabindex="0"] span').contains('Multi-configuration project').click()
+        cy.get('#ok-button').click()
+        cy.get('.jenkins-breadcrumbs__list-item:nth-child(1) > .model-link').click()
+
+        cy.get('td a[href*="job"].jenkins-table__link').should('have.text', 'Project1')
```
Comment (Rom-T) [inline cypress/e2e/multiconfigProject1.cy.js:11]: Since we already know that we need to store data in fixtures, please transfer `Project1` to fixtures.

**Pull request closed without merging**